src/app/pages/ArticlePage/index.test.jsx

```diff
@@ -218,3 +220,97 @@ it('should render a ltr article (pidgin) with most read correctly', async () =>
   expect(mostReadSection).not.toBeNull();
   expect(container).toMatchSnapshot();
 });
+it('should focus on id when anchor link is clicked', async () => {
+  const articleDataNewsWithSummary = mergeDeepLeft(
+    {
+      content: {
+        model: {
+          blocks: [
+            {
+              id: '1',
+              type: 'text',
+              model: {
+                blocks: [
+                  {
+                    id: '2',
+                    type: 'paragraph',
+                    model: {
+                      text: 'Anchor 1',
+                      blocks: [
+                        {
+                          id: '3',
+                          type: 'urlLink',
+                          model: {
+                            text: 'Anchor 1',
+                            blocks: [
+                              {
+                                id: '4',
+                                type: 'fragment',
+                                model: {
+                                  text: 'Anchor 1',
+                                  attributes: ['bold'],
+                                },
+                              },
+                            ],
+                            locator:
+                              'https://www.test.bbc.com/mundo/articles/ce4krqk1334o#Anchor-1',
+                            isExternal: false,
+                          },
+                        },
+                      ],
+                    },
+                  },
+                ],
+              },
+            },
+            {
+              id: '5',
+              type: 'subheadline',
+              model: {
+                blocks: [
+                  {
+                    id: '6',
+                    type: 'text',
+                    model: {
+                      blocks: [
+                        {
+                          id: '7',
+                          type: 'paragraph',
+                          model: {
+                            text: 'Anchor 1',
+                            blocks: [
+                              {
+                                id: '8',
+                                type: 'fragment',
+                                model: {
+                                  text: 'Anchor 1',
+                                  attributes: [],
+                                },
+                              },
+                            ],
+                          },
+                        },
+                      ],
+                    },
+                  },
+                ],
+              },
+            },
+          ],
+        },
+      },
+    },
+    articleDataNews,
+  );
+  const { container } = render(
+    <Context service="news">
+      <ArticlePage pageData={articleDataNewsWithSummary} />
+    </Context>,
+  );
+  const link = container.querySelector('a');
```

Comment:
Can we make use of react-testing-library queries instead e.g. getByText

Reply:
Great suggestion. Done!